src/components/QuestDetails/index.module.scss

```diff
@@ -59,3 +59,13 @@
   height: 100%;
   gap: var(--space-xl);
 }
+
+.backButton {
+  width: var(--space-xl-fixed);
+  height: var(--space-xl-fixed);
+  min-width: var(--space-xl-fixed);
+  min-height: var(--space-xl-fixed);
+  padding: var(--space-xs);
+  border-radius: 100%;
+  border: 1px solid var(--color-neutral-100);
```

Comment:
let's use the default color styles for circular button for consistency across the app